### PR TITLE
Incrementalified

### PR DESCRIPTION
**Pull request by jglick**
Extracted from #229, since I also expect to need incremental versions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,12 +28,12 @@
...
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.4</version>
+        <version>3.13</version>
         <relativePath />
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-cps</artifactId>
-    <version>2.54-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```
Comment (abayer) [inline pom.xml]: Because I haven't been paying enough attention - how does this work with `mvn release:prepare release:perform`?

Comment (jglick) [reply pom.xml]: [Guide](https://github.com/jenkinsci/incrementals-tools/blob/master/README.md#running-maven-releases)

File: src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java [hunk 2]
```diff
@@ -535,7 +535,7 @@ public HttpResponse doGenerateSnippet(StaplerRequest req, @QueryParameter String
         return ExtensionList.lookup(SnippetizerLink.class);
     }
 
-    @Restricted(DoNotUse.class)
+    @Restricted(NoExternalUse.class)
```
Comment (svanoort) [inline src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java]: Seems unrelated?

**Comment by svanoort** [comment]
You have the reviews, can you just merge+release the plugin pom version rather than using a special version?

File: src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java (hunk 2 above)
Comment (jglick) [reply src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java]: Mandated by updating the parent POM; works around https://github.com/kohsuke/access-modifier/pull/12.

**Comment by jglick** [comment]
> can you just merge+release the plugin pom version

Will do.
